gatsby/src/components/BackToTop/BackToTop.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState, useCallback } from "react";
+import ChevronUp from "@nice-digital/icons/lib/ChevronUp";
+
+import styles from "./BackToTop.module.scss";
+
+interface BackToTopProps {
+	scrollTargetId?: string;
+}
+
+export const BackToTop: React.FC<BackToTopProps> = ({
+	scrollTargetId: scrollTo = "content-start",
+}: BackToTopProps) => {
+	const [isFixed, setIsFixed] = useState(false);
+
+	const handleScroll = useCallback(() => {
+		const footer: Element | null = document.querySelector("footer");
+
+		if (!footer) return;
+
+		const footerVisible: boolean =
+			window.scrollY + window.innerHeight >
+			document.body.scrollHeight - footer.clientHeight;
+
+		const scrolledDown: boolean = window.scrollY > 800;
+
+		setIsFixed(!footerVisible && scrolledDown);
+	}, []);
+
+	useEffect(() => {
+		document.addEventListener("scroll", handleScroll);
+
+		return () => {
+			document.removeEventListener("scroll", handleScroll);
+		};
+	}, []);
+
+	return (
+		<div
+			role="navigation"
```

Comment:
I'm not convinced by `role="navigation"` - either it's a proper `nav` element so implicitly has `role="navigation"` or it's just not. I'd favour the second because if you're using a screen reader to scan landmarks etc, then you don't need to find a back to top  one....as you can scan headers go to the main landmark, so don't think it's needed. Which also means we wouldn't need that extra aria label.

Reply:
agreed, felt a bit hacky. was trying to avoid the 'every element needs a containing landmark' warnings